zerver/views/thumbnail.py

```diff
@@ -33,14 +17,17 @@ def backend_serve_thumbnail(
     url: str,
     size: str,
 ) -> HttpResponse:
-    if not maybe_user_profile.is_authenticated:
-        realm = get_valid_realm_from_request(request)
-    else:
-        assert isinstance(maybe_user_profile, UserProfile)
-        realm = maybe_user_profile.realm
-
-    if not validate_thumbnail_request(realm, maybe_user_profile, url):
-        return HttpResponseForbidden(_("<p>You are not authorized to view this file.</p>"))
-
-    thumbnail_url = generate_thumbnail_url(url)
-    return redirect(thumbnail_url)
+    # This URL used to be passed arbitrary URLs, and pass them through
+    # Camo; we no longer support doing so, and instead return a
+    # redirect to those locations.
```

Comment:
Probably worth extending this to talk about how this endpoint is unused in modern Zulip but is retained for backwards-compatibilty?